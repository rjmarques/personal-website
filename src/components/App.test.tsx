// import { shallow, ShallowWrapper } from "enzyme";
// import * as React from "react";
// import * as renderer from "react-test-renderer";
// import App, { IAppProps } from "./App";

describe("renders without crashing", () => {
  // let app: ShallowWrapper<undefined, undefined>;
  // const props: IAppProps = {
  //   message: "This is a test message!"
  // };
  // beforeEach(() => {
  //   app = shallow(<App {...props} />);
  // });
  // it("renders correctly", () => {
  //   const tree = renderer.create(<App {...props} />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
  // it("should render h1", () => {
  //   const header = app.find("h1");
  //   expect(header.text()).toBe(props.message);
  // });
});
