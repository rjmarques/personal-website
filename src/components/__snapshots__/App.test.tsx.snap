// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing renders correctly 1`] = `
<div
  className="App"
>
  <h1
    className="App-header"
  >
    This is a test message!
  </h1>
  <button
    className="ant-btn ant-btn-primary"
    onClick={[Function]}
    title={undefined}
    type="button"
  >
    <span>
      Primary
    </span>
  </button>
</div>
`;
